gcd.py

```diff
@@ -0,0 +1,9 @@
+#! /usr/bin/env python3
+
+
+def gcd(args):
```

Comment:
Is this different from https://stackoverflow.com/questions/11175131/code-for-greatest-common-divisor-in-python ?  If so, add some comments.